### DISCUSSION TITLE
出现这个是什么问题？

### DISCUSSION THREAD

**Discussion by emm20060814** [mikumifa/biliTickerBuy#320] [提问专区]
![wx_camera_1719806621858](https://github.com/mikumifa/biliTickerBuy/assets/174272822/4ee0095c-b8c5-4ddd-8ee4-2d09d7b4910c)

**Comment by CinnabarCHU** [comment]
我经常遇到这个问题，主要发生在kernel重启但是前端没刷新的时候，推测是kernel回传给Web GUI更新网页时出了问题，卡死了，如果是Python启动的可以刷新网页，如果是WIndows分发版直接重启吧

**Reply by emm20060814** [reply]
谢谢！

**Reply by gpc123456** [reply]
您是不是使用了手动过验证码，如果是的话，需要看一下gradio的前端页面最下方，看看是不是弹出了手动验证码，手动验证的话需要填写完验证码后再点右侧的`完成验证码后点此按钮`才能够完成验证。

此外，如果您重启了kernel，请务必刷新一次前端页面来与kernel建立新的连接，旧连接可能会出现问题。
